browser/brave_wayback_machine/wayback_machine_url_fetcher.cc

```diff
@@ -0,0 +1,89 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_wayback_machine/wayback_machine_url_fetcher.h"
+
+#include <utility>
+
+#include "base/bind.h"
+#include "base/json/json_reader.h"
+#include "net/traffic_annotation/network_traffic_annotation.h"
+#include "services/network/public/cpp/resource_request.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+#include "url/gurl.h"
+
+namespace {
+
+constexpr char kWaybackQueryURL[] =
+    "https://archive.org/wayback/available?url=";
```

Comment:
should this be in a constants file? ex: `common/url_constants.cc`

Reply:
Yes, `url_constants.cc` looks better place for this `url constant`. :)